### ISSUE TITLE
Angular Integration

### ISSUE THREAD

**Issue by mohitmg25** [ether/etherpad-lite#3998]
I am trying to integrate Etherpad real time collaborative editor on my Angular project.
So anybody can help in guiding me?
Thanks in advance

**Comment by JohnMcLear** [comment]
As a rule of thumb, we don't encourage questions like this.  "Introduce X service into Y Framework."  The question should be asked on Y Framework how to integrate X service.  We don't support every Framework that comes around, the technical costs are just too high.  Sorry but I don't want this to be a burden for our community but if someone wants to run it as a project I wouldn't be against debating the benefits of X framework over our current solution.

Eitherpad is pretty much a single page application so for the most parts frameworks are overkill but if they remove our technical debt of iframes / javascript in the html etc. then I'd be up for that :)

The problem is if we have the issue/debate about Angular we have to consider React et al and unless someone is going to put a lot of technical effort into fixing the warts I don't see this being easy to do.

So TLDR; if you want to put Etherpad in your project, put it in an iFrame and leave Etherpad to do it's thing.  Etherpad is not some little plugin or module you just import to your hipster Framework, don't think it is.  Sorry again, just setting expectations really early on as to not go down a rabbit hole.
